test/access-control.integration.js

```diff
@@ -111,9 +111,15 @@ describe('access control - integration', function() {
           assert.equal(user.password, undefined);
         });
       });
+
+      // user has replaceOnPUT = false; so then both PUT and PATCH should be allowed for update
       lt.describe.whenCalledRemotely('PUT', '/api/users/:id', function() {
```

Comment:
Oh, only now I realised that you are explicitly setting `replaceOnPUT=false` for `User` model. In that case, I think we need to add a test case verifying that `POST /api/users/:id/replace` is allowed.